readers/api/requests.go

```diff
@@ -39,5 +41,47 @@ func (req listMessagesReq) validate() error {
 		return apiutil.ErrInvalidComparator
 	}
 
+	validAggregations := map[string]bool{
```

Comment:
we have 5 aggregation functions: https://www.simplilearn.com/tutorials/sql-tutorial/sql-aggregate-functions#:~:text=An%20aggregate%20function%20in%20SQL%20performs%20a%20calculation%20on%20multiple,except%20for%20the%20count%20function.